src/audio/drc/drc_math_lib/asin/c_fixpt/drc_asin.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * File: drc_asin.h
+ *
+ * Author :Shriram Shastry
+ * Copyright (c) 2016, Intel Corporation All rights reserved.
+ */
+
+#ifndef DRC_ASIN_H
+#define DRC_ASIN_H
+
+/* Include Files */
+#include <stddef.h>
+#include <stdlib.h>
+#include "typedef.h"
+#include "drc_asin_types.h"
+
+/* Function Declarations */
+extern void drc_asin_fixpt(const signed char x[2], int y[2]);
+extern void drc_asin_initialize(void);
+extern void drc_asin_terminate(void);
+extern void init_struc_fixpt(signed char x[2]);
```

Comment:
What do the above three (initialize, terminate, init_struc) need for?

Reply:
We should avoid extern if possible as well, what is the benefit of using that here?

Reply:
I will remove initialize, terminate, function calls except  init_struc_fixpt(). function  init_struc_fixpt is initializing the testvector for library function.